### DISCUSSION TITLE
ItemStack.Builder from ItemStack?

### DISCUSSION THREAD

**Discussion by KyllianGamer** [Minestom/Minestom#1644] [Help]
When given an ItemStack, is it possible to create a Builder object out of it?

**Comment by KyllianGamer** [comment]
Ok, nevermind. I just remembered that I can just create a new Builder object and just copy over the material and meta.

**Reply by Tofaa2** [reply]
Why do you need a builder for ItemStack considering it is all immutable and can be just recycled

like
`java
final ItemStack item = ItemStack.of(Material.WOODEN_SWORD).withAmount(10).withLore(lore) etc etc
`
